app/backend/src/tests/test_fixtures.py

```diff
@@ -44,48 +46,82 @@ def db():
     return sessionmaker(bind=engine)
 
 
-def generate_user(db, username=None):
+def _raw_generate_user(session, username, jailed):
+    """
+    Internal user creation code
+    """
+    if not username:
+        username = "test_user_" + random_hex(16)
+
+    user = User(
+        username=username,
+        email=f"{username}@dev.couchers.org",
+        # password is just 'password'
+        # this is hardcoded because the password is slow to hash (so would slow down tests otherwise)
+        hashed_password=b"$argon2id$v=19$m=65536,t=2,p=1$4cjGg1bRaZ10k+7XbIDmFg$tZG7JaLrkfyfO7cS233ocq7P8rf3znXR7SAfUt34kJg",
+        name=username.capitalize(),
+        city="Testing city",
+        verification=0.5,
+        community_standing=0.5,
+        birthdate=date(year=2000, month=1, day=1),
+        gender="N/A",
+        languages="Testing language 1|Testing language 2",
+        occupation="Tester",
+        about_me="I test things",
+        about_place="My place has a lot of testing paraphenelia",
+        countries_visited="Testing country",
+        countries_lived="Wonderland",
+        # you need to make sure to update this logic to make sure the user is jailed/not on request
+        accepted_tos=0 if jailed else 1,
```

Comment:
This could make testing of other jailing reasons difficult. Maybe for now you could change the argument from 'jailed' to 'acceptedTOS'?

Reply:
Actually, on a second thought, I don't think this should be a parameter here anyway, the jail test modifies the user as needed anyway. I'll refactor it out.